config/initializers/cellect.rb

```diff
@@ -6,10 +6,6 @@ def self.cellect_on
   def self.cellect_min_pool_size
     @cellect_min_pool_size ||= (ENV["CELLECT_MIN_POOL_SIZE"] || 10000).to_i
   end
-
-  def self.use_cellect?(workflow)
-    cellect_on && workflow.using_cellect?
```

Comment:
we should probably clean up the feature flag `self.cellect_on` as well if we remove this